lib/bolt_ext/server.rb

```diff
@@ -6,6 +6,10 @@
 require 'json'
 
 class TransportAPI < Sinatra::Base
+  get '/' do
```

Comment:
Is this for testing?

Reply:
Ah, yes it is :)

Reply:
It also gives us a simple API to hit to see that the server's running. It's usually nice when the root path returns 200.